### PR TITLE
Feature/preact test

### PR DESCRIPTION
**Pull request by spacebeers**
Let's test the test. Leave any feedback here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,123 +1,38 @@
...
 # Virgin Atlantic ~ Hotel Search Results (Front-end)
 ## Rules
-You must write the application in React or Preact (as this is our technology of choice).
+You must write the application in [Preact](https://preactjs.com/) and [TypeScript](https://www.typescriptlang.org/). Both are set up and ready to use inside this project.
 
 In addition you should adhere to the following conditions:
 
-1) It must be easy to run using libraries & tools commonly available on a development machine. 
-1) Clear instructions for how to build and run the application should be included within the code, e.g. `README.md`
 1) The code must be your own work. If you have a strong case to use a small code snippet of someone else's e.g. a
```
Comment (monkeychops) [inline README.md:7]: This should be 1), 2), 3) etc

**Comment by spacebeers** [comment]
Looking good to me...

File: README.md [hunk 2]
```diff
@@ -1,123 +1,38 @@
...
 
 ## What it should do
-Build a 'search results page' which connects to our holiday search API to display a list of holidays for a given location and departure date.  Select what data items (example listed below) you think should be included on the page.
-
-Available locations include: 'new-york', 'orlando', 'barbados', 'toronto'.
+Build the 'search results page' which connects to our holiday search API to display a list of holidays for a given location and departure date. Select what data items (example listed below) you think should be included on the page.
 
 Add the ability to filter the results further by:
 
 1) Price per person
```
Comment (monkeychops) [inline README.md:19]: 1, 2, 3 ...

File: README.md (hunk 1 above)
Comment (spacebeers) [reply README.md:7]: that's just a `<ol> `in Markdown

File: src/routes/results.route.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    "adults": searchParams?.matches?.adults as unknown as number,
+                    "childAges": [],
+                    "infants": 0
+                }
+            ]
+        }
+
+        doRequest('POST', '/cjs-search-api/search', requestBody)
+            .then((response: BookingResponse) => {
+                console.info(response)
```
Comment (monkeychops) [inline src/routes/results.route.tsx]: Was thinking we should leave this bit out

File: src/routes/results.route.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+
+    return (
+        <section>
+            <SearchComponent />
+
+            <h1>Results should display here.</h1>
+
+            {
+                loaded ? "Results loaded ✅" : "Results pending..."
```
Comment (monkeychops) [inline src/routes/results.route.tsx]: And maybe leave this too

**Review by monkeychops — COMMENTED** [review]
👍

**Pull request merged**